importer/wdi_importer.py

```diff
@@ -1,63 +1,922 @@
+                    if cell.value:
+                        global_cat[indicator]['unitofmeasure'] = cell.value
+                    else:
+                        if '(' not in global_cat[indicator]['name']:
+                            global_cat[indicator]['unitofmeasure'] = ''
+                        else:
+                            global_cat[indicator]['unitofmeasure'] = global_cat[indicator]['name'][global_cat[indicator]['name'].index('('):-1].replace('(', '').replace(')','')
+                if column_number == 11:
+                    if cell.value:
+                        global_cat[indicator]['limitations'] = cell.value
+                    else:
+                        global_cat[indicator]['limitations'] = ''
+                if column_number == 12:
+                    if cell.value:
+                        global_cat[indicator]['sourcenotes'] = cell.value
+                    else:
+                        global_cat[indicator]['sourcenotes'] = ''
+                if column_number == 13:
+                    if cell.value:
+                        global_cat[indicator]['comments'] = cell.value
+                    else:
+                        global_cat[indicator]['comments'] = ''
+                if column_number == 14:
+                    global_cat[indicator]['source'] = cell.value
+                if column_number == 15:
+                    if cell.value:
+                        global_cat[indicator]['concept'] = cell.value
+                    else:
+                        global_cat[indicator]['concept'] = ''
+                if column_number == 17:
+                    if cell.value:
+                        global_cat[indicator]['sourcelinks'] = cell.value
+                    else:
+                        global_cat[indicator]['sourcelinks'] = ''
+                if column_number == 18:
+                    if cell.value:
+                        global_cat[indicator]['weblinks'] = cell.value
+                    else:
+                        global_cat[indicator]['weblinks'] = ''
+                global_cat[indicator]['saved'] = False
+
+        column_number = 0
+
+    category_vars = {}  # categories and their corresponding variables
+
+    for key, value in global_cat.items():
+        if category_vars.get(value['category'], 0):
+            category_vars[value['category']].append(key)
+        else:
+            category_vars[value['category']] = []
+            category_vars[value['category']].append(key)
+
+    existing_categories = DatasetCategory.objects.values('name')
+    existing_categories_list = []
+
+    for each in existing_categories:
+        existing_categories_list.append(each['name'])
+
+    if wdi_category_name_in_db not in existing_categories_list:
+        the_category = DatasetCategory(name=wdi_category_name_in_db)
+        the_category.save()
+        backup_sql_file.write("INSERT INTO %s (name, created_at, updated_at) VALUES ('%s', '%s', '%s');\n" %
```

Comment:
I'm not sure I understand the purpose of having a separate backup file. If we're writing out all of the statements in full raw SQL anyway, why not just generate this sql file and then run that for the actual import?